src/components/autocomplete/js/autocompleteController.js

```diff
@@ -639,7 +679,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   /**
    * Clears the searchText value and selected item.
    */
-  function clearValue ($event) {
+  function clearValue (shouldClearText) {
```

Comment:
Maybe I wasn't clear. That's the problem though. `clearValue` will always clear the selected item and the text. I'm trying to allow text clearing to be optional.